pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration.go

```diff
@@ -227,9 +250,130 @@ func (s *Service) UpdateMigration(ctx context.Context, id int64, cm cloudmigrati
 	return nil, nil
 }
 
-func (s *Service) RunMigration(ctx context.Context, uid string) (*cloudmigration.CloudMigrationRun, error) {
-	// TODO: Implement method
-	return nil, nil
+func (s *Service) GetMigrationDataJSON(ctx context.Context, id int64) ([]byte, error) {
+	var migrationDataSlice []cloudmigration.MigrateDataRequestItemDTO
+	// Data sources
+	dataSources, err := s.getDataSources(ctx, id)
+	if err != nil {
+		s.log.Error("Failed to get datasources", "err", err)
+		return nil, err
+	}
+	migrationDataSlice = append(migrationDataSlice, cloudmigration.MigrateDataRequestItemDTO{
```

Comment:
This needs to be refactored slightly to be one item per data source (and same with dashboard+folder below). The refId, name, and data should refer to the datasource uuid, name, and object.